README.md

```diff
@@ -76,7 +74,6 @@ future:
 - `jsdocTypeVisitorKeys` - [VisitorKeys](https://github.com/eslint/eslint-visitor-keys)
     for `jsdoc-type-pratt-parser`.
 - `getTokenizers` - A utility. Might be removed in future.
-- `toCamelCase` - A utility. Might be removed in future.
 - `hasSeeWithLink` - A utility to detect if a tag is `@see` and has a `@link`
```

Comment:
And this duplicated line?

Reply:
I assume you are referring to the duplicated `getTokenizers` / `hasSeeWithLink` lines that previously existed in the README before my changes. I also removed any additional "This might be removed in the future" references.

If there is any more general "housekeeping" tasks beyond the scope of this PR that you'd like to complete it may be easiest to do so yourself after the PR is merged.

Reply:
That's great, thanks.